stringtie.wdl

```diff
@@ -10,6 +10,7 @@ task Stringtie {
 
     command {
         set -e -o pipefail
+        mkdir -p ${sub(assembledTranscriptsFile, basename(assembledTranscriptsFile), "")}
```

Comment:
`mkdir -p $(dirname ${assembledTranscriptsFile})` gives the same result. (Yes, `dirname` exists! :tada: )